editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/util/camel/CamelFacetVersionChangeDelegate.java

```diff
@@ -85,8 +87,12 @@ private void updateCamelVersion(IProject project, String camelVersion) throws Co
 		// TODO: this block ensures that we have the staging repo for camel 2.17 in our pom.xml
 		// so we can find that unreleased camel version. this becomes obsolete once the camel version 
 		// has been released and can be disabled / removed / used for a new unreleased camel version 
-		MavenUtils.ensureRepositoryExists(m2m.getRepositories(), CAMEL_STAGING_REPO_URI, "camelStaging");
-		MavenUtils.ensureRepositoryExists(m2m.getPluginRepositories(), CAMEL_STAGING_REPO_URI, "camelStaging");
+		MavenUtils.ensureRepositoryExists(m2m.getRepositories(), PRODUCT_STAGING_REPO_URI, "fuse-ea");
+		MavenUtils.ensureRepositoryExists(m2m.getPluginRepositories(), PRODUCT_STAGING_REPO_URI, "fuse-ea");
+		MavenUtils.ensureRepositoryExists(m2m.getRepositories(), THIRD_PARTY_STAGING_REPO_URI, "redhat-ea-repository");

```

Comment:
these repo entries are only needed as long as we are non-GA. After the GA these lines are not needed any more.